README.md

```diff
@@ -1,3 +1,55 @@
+# icb-website-react
+
+## Setup
+
+- Install npm
+
+- TODO
+
+## Workflow
+
+**You MUST be familiar with pull requests and git basics before you can work on this project.**
+
+> We will use git as our version-control system. If you don't know what git is, read more [here](https://www.freecodecamp.org/news/learn-the-basics-of-git-in-under-10-minutes-da548267cc91/).
+
+> We will be using pull-requests to manage this project. If you do not know what a pull-request is, read more [here](https://learn.co/lessons/github-pull-request-basics).
+
+If you have any questions about this process, please use the internet as a resource. If that doesn't work please ask for help from someone who knows how this stuff works.
+
+### Getting started
+
+- If you have never worked on the project before, start by cloning the repository onto your local machine.
+  - In your terminal on your computer, navigate to the directory (folder) where you want the project to live
+  - In this directory, run the command `git clone https://github.com/inventioncorps/icb-website-react.git icb-website`
+
+### Contributing
+
+> Visual Studio Code and other code editors have terminals embedded in their UI. It's usually easier to work from within the code editor for these kinds of things - otherwise you have to keep switching back and forth between the code itself and the terminal.
+
+- To start working on a new feature:
+
+  - Make sure you are on the `develop` branch as this will have the latest changes to the codebase. You can run `git checkout develop` to switch to the branch if you are not already there.
+  - Create a new branch called `feature\<name>-<description>` by running the command `git branch feature\<name>-<description>`. Replace`<name>` with your name and `<description>` with a short description of the feature. For example, if Bongi was working on the README he might use the branch name `feature\bongi-readme`.
+  - Checkout (switch to) that branch by running `git checkout feature\<name>-<description>`.
+  - Make sure the remote repository is tracking your local branch. Run `git push -u origin feature\<name>-<description>`
```

Comment:
This step requires an auth token. Maybe include some instructions on obtaining that

Reply:
Is this the github authentication process for users or something else?

Let's go over this together if possible.